src/Raven.Server/Utils/MicrosoftLogging/SparrowLoggerWrapper.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Microsoft.Extensions.Logging;
+using Sparrow.Logging;
+
+namespace Raven.Server.Utils.MicrosoftLogging;
+
+public class SparrowLoggerWrapper<TLogger> : ILogger<TLogger>
+{
+    private readonly Logger _sparrowLogger;
+
+    public SparrowLoggerWrapper(Logger sparrowLogger)
+    {
+        _sparrowLogger = sparrowLogger;
+    }
+
+    public IDisposable BeginScope<TState>(TState state) where TState : notnull
+    {
+        return NullScope.Instance;
+    }
+
+    public bool IsEnabled(LogLevel logLevel) => true;
+
+    public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+    {
+        if (formatter == null)
+            throw new ArgumentNullException(nameof(formatter));
+        
+        var logLine = formatter(state, null);
+        _sparrowLogger.Operations(logLine, exception);
```

Comment:
is this logger turned on always for Operations? Is there a way to know if the log was from Critical lvl or Debug?

Reply:
Yes, I leave it for "Operation" because our levels do not match Microsoft levels.
Changing that may require too many changes to the logging source code that is shared with the client code.
But I forgot to add the Microsoft level.
Adding it now

```
Time, Thread, Level, Source, Logger, Message, Exception
2022-12-21T06:30:54.4184469Z, 1, Operations, Microsoft.AspNetCore.Hosting, Diagnostics, Debug, Hosting starting
2022-12-21T06:30:54.8193468Z, 1, Operations, Microsoft.AspNetCore.Hosting, Diagnostics, Debug, Hosting started
2022-12-21T06:30:55.1581871Z, 1, Operations, Microsoft.AspNetCore.Hosting, Diagnostics, Debug, Loaded hosting startup assembly Raven.Server
2022-12-21T06:32:07.7080417Z, 16, Operations, Microsoft.AspNetCore.Server.Kestrel, Connections, Debug, Connection id "0HMN33K9VR4VE" accepted.
2022-12-21T06:32:08.2911816Z, 13, Operations, Microsoft.AspNetCore.Server.Kestrel, Connections, Debug, Connection id "0HMN33K9VR4VE" started.
2022-12-21T06:32:08.8539116Z, 13, Operations, Microsoft.AspNetCore.Hosting, Diagnostics, Information, Request starting HTTP/1.1 GET http://localhost:8080/setup/alive - -
2022-12-21T06:32:09.3138279Z, 13, Operations, Microsoft.AspNetCore.ResponseCompression, ResponseCompressionProvider, Trace, This request accepts compression.
2022-12-21T06:32:09.7660004Z, 13, Operations, Microsoft.AspNetCore.Server.Kestrel, Connections, Debug, Connection id "0HMN33K9VR4VE" completed keep alive response.
2022-12-21T06:32:10.1045619Z, 13, Operations, Microsoft.AspNetCore.Hosting, Diagnostics, Information, Request finished HTTP/1.1 GET http://localhost:8080/setup/alive - - - 204 - - 1429.2636ms
```